### PR TITLE
Extend quality list

### PR DESCRIPTION
**Pull request by mweidling**
I added some aspects to the software quality list I regard as important, but which can of course be discussed. Maybe we should think about restructuring the list, since the current structure doesn't seem handy to me -- especially the "General" part reads more like a "Miscellaneous" part. The divisions of DIN ISO 9126 might be worth a look.

### CODE REVIEW

File: quality/software-checklist.md [hunk 1]
```diff
@@ -5,26 +5,34 @@
 - [ ] Does the software have a descriptive name?
 - [ ] Is there a short high-level description of the software?
 - [ ] Is the purpose of the software clear?
+- [ ] Does the software exactly match its requirements?
 - [ ] Is the targeted audience of the software clear?
-- [ ] Does it (and its dependencies) use OSI approved licenses?
+- [ ] Has the software been tested by members of the target audience in respect of its usability?
```
Comment (yoannspace) [inline quality/software-checklist.md:10]: Do we want this check in here? Not saying testing it by the target audience is not a good idea, but should that affect the quality of the software?

File: quality/software-checklist.md [hunk 2]
```diff
@@ -5,26 +5,34 @@
...
 - [ ] Is there a website for the software?
+- [ ] Is the software's website mobile friendly to a certain degree? I.e. can it be accessed on a
+  smartphone or tablet without hiding the most important information and features?
+- [ ] Are the user interface design and the software's website mindful of accessibility? I.e. does it
+  consider e.g. a high contrast between colors for colorblind users, are there alternative texts for
+  images that can be read by a screenreader, are texts easily resizeable, ...?  
 - [ ] Does the software have a release mechanism?
 - [ ] Is the software available in packaged format or only sources?
-- [ ] Are maintainer and development status clear, including contact information?
+- [ ] Are maintainer and development status clear, including up to date and accessible contact information?
```
Comment (yoannspace) [inline quality/software-checklist.md:21]: Up to date until time of writing but I don't think we can expect all these information to always be up to date.

**Review by yoannspace — CHANGES_REQUESTED** [review]
2 comments to be cleared up before possible merge

**Comment by schildwaechter** [comment]
Please rebase against the new sphinx solution.

File: quality/software-checklist.md (hunk 1 above)
Comment (mweidling) [reply quality/software-checklist.md:10]: I thought about this some time and in my opinion this is important. Software can be efficient, correct etc., but when it's not usable, it probably won't be accepted - or used.

File: quality/software-checklist.md (hunk 2 above)
Comment (mweidling) [reply quality/software-checklist.md:21]: Which scenarios do you have in mind where this is not possible?

Comment (yoannspace) [reply quality/software-checklist.md:21]: I mean tools or libraries still being used but not maintained so no contact exists, but it is very likely implied so I will accept it as such.

**Pull request closed without merging**